Extension/src/abTesting.ts

```diff
@@ -61,6 +67,12 @@ export class ABTestSettings {
         return util.isNumber(this.settings.gotoDefIntelliSense) ? this.settings.gotoDefIntelliSense >= this.bucket.Value : true;
     }
 
+    public getMinimumVSCodeVersion(): PackageVersion {
+        // Get minimum VS Code's supported version for latest insiders upgrades.
+        return new PackageVersion(
+            this.settings.minimumVSCodeVersion ? this.settings.minimumVSCodeVersion : this.minimumVSCodeVersionDefault.Value);
```

Comment:
`this.settings.minimumVSCodeVersion` should always be valid, so there's no need for the `?` anymore.

Reply:
Yes, but the same logic applies to other members of `Settings`, like `this.settings.gotoDefIntelliSense`. 
I needed to define `Settings `like this: 
```
interface Settings {
    defaultIntelliSenseEngine?: number;
    recursiveIncludes?: number;
    gotoDefIntelliSense?: number;
    enhancedColorization?: number;
    minimumVSCodeVersion**?**: string;
}
```
because I use this interface to read the `Settings `from the JSON file, and the `minimumVSCodeVersion `might not exists in the JSON file.